recipe/conda_build_config.yaml

```diff
@@ -23,5 +23,5 @@ enable_cuda:
   - False     # [not (linux64 or ppc64le or aarch64)]
 
 # Needed for MAP_ANONYMOUS and clock_gettime:
-MACOSX_DEPLOYMENT_TARGET:        # [osx and x86_64]
-  - "10.13"                      # [osx and x86_64]
+c_stdlib_version:              # [osx and x86_64]
+  - 10.13                      # [osx and x86_64]
```

Comment:
```suggestion
  - "10.13"                      # [osx and x86_64]
```
(Otherwise this is a floating point number and subject to rounding error.)

Reply:
`10.13` is what they have here: https://conda-forge.org/news/2024/03/24/stdlib-migration/

Reply:
:shrug: If it accepts the string, I'd say that's better.

Reply:
Added in 9bfb1df. Both ways seem to render exactly the same .